R/spec-result-roundtrip.R

```diff
@@ -582,8 +582,8 @@ has_utf8_or_ascii_encoding <- function(x) {
     FALSE
 }
 
-is_raw_list <- function(x) {
-  is.list(x) && is.raw(x[[1L]])
+is_blob <- function(x) {
```

Comment:
I'd rather test that the return value is coercible to a blob. It puzzles me that the tests pass, because none of the backends currently return the blob type. (Travis checks the rstats-db backends in a custom step).

Could you please run `make --keep-going --directory=revdep-dev` in your clone, and check if this code is touched at all?